indico/web/flask/app.py

```diff
@@ -120,6 +120,7 @@ def configure_multipass(app, config):
     app.config['MULTIPASS_LOGIN_URLS'] = None  # registered in a blueprint
     app.config['MULTIPASS_SUCCESS_ENDPOINT'] = 'categories.display'
     app.config['MULTIPASS_FAILURE_MESSAGE'] = _('Login failed: {error}')
+    app.config['MULTIPASS_HIDE_NO_SUCH_USER'] = True
```

Comment:
I don't really like the idea of unconditionally setting this... IMHO it should be up to the admin if they prefer user-friendly errors, or fail with a generic message.

Reply:
Would you, then, expect that this is asked in the CLI wizard?

Reply:
not sure if it's that important... I think people who care about such things will probably go through the available config options anyway and set them accordingly

Reply:
The question is... What should be the behavior if the key is not explicitly defined in `indico.conf`?

Reply:
> Or should Indico fail to start if this key is not set?

Certainly not

>  What should be the behavior if the key is not explicitly defined in indico.conf? 

I'd say since typically knowing whether an email/username on Indico exists or not is not particularly sensitive, I'd default to the current verbose behavior.